test/indexer/test_tdr.py

```diff
@@ -180,6 +180,23 @@ def test_fetch_bundle(self):
         with self.assertRaises(RequirementError):
             self._test_fetch_bundle(bundle, load_tables=False)
 
+    def test_disallow_double_pipe_joiner(self):
```

Comment:
```suggestion
    def test_disallow_double_pipe_joiner(self):
```
```suggestion
    def test_disallow_manifest_column_joiner(self):
```

Reply:
I'm not convinced `test_tdr` is the right place for this, but I can't think of a better option at the moment.